docs/guides/operator/installation.adoc

```diff
@@ -21,6 +21,66 @@ The recommended way to install the {project_name} Operator in Kubernetes environ
 
 * Be sure that you have cluster-admin permission or an equivalent level of permissions granted by an administrator.
 
+[WARNING]
+====
+*Important: Automatic OLM Upgrades*
+
+By default, OLM automatically updates the {project_name} Operator when a new version is released. This can cause several significant issues:
+
+* When using the default {project_name} image, the Operator by default uses a matching image of the corresponding {project_name} version, resulting in *unintended {project_name} upgrades* when the Operator is upgraded.
+
+* Even when using custom images, major Operator upgrades can introduce compatibility issues with your existing Keycloak CR configuration.
+
+*Potential risks:*
+
+* Unexpected downtime during automatic upgrades
+* Database schema migrations that cannot be easily rolled back
+* Version incompatibilities if custom resources use features from newer CRDs
+* Breaking changes in the Operator behavior across major versions
+
+*Recommendation:*
+
+*We strongly recommend using only manual approval mode for the Keycloak Operator.* This ensures you can:
+
+1. Schedule maintenance windows for upgrades
+2. Test upgrades in a non-production environment first
+3. Prepare rollback plans in case of issues
```

Comment:
May need to elaborate here - or at least link to the general recommendations on changing versions, which mention backing up the database.

Reply:
We do not have specific backup instructions in the [upgrading guide](https://www.keycloak.org/docs/latest/upgrading/index.html#_upgrading). Still, we might improve the wording, something like:
```suggestion
3. Back up the database to allow downgrading to the previous {project_version} in case of issues
```

Reply:
@shawkins do you have specific section/guide to link here in mind?